SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -588,7 +588,14 @@ Worker_ComponentId USpatialClassInfoManager::ComputeActorInterestComponentId(con
 
 	if (ActorForRelevancy->bAlwaysRelevant)
 	{
-		return SpatialConstants::ALWAYS_RELEVANT_COMPONENT_ID;
+		if (ActorForRelevancy->GetClass()->HasAnySpatialClassFlags(SPATIALCLASS_ServerOnly))
```

Comment:
I would suggest maybe wrapping this in a function on the actor. Further specifying a flag via a different api seems a bit brittle (it will be an easy mistake to check one and not the other).
Something like
```
Relevance AActor::getRelevance()
```
This would adding another enum into actor class that is matched against its SpatialConstant counter part here.

Reply:
This functionality is only required at this point - and adding it to the native unreal actor interface wouldn't be desired.

What we're essentially asking here is, for this actor, what is the interest component that we should be adding to it's entity data to allow the appropriate workers to have interest.